src/backend/utils/__mocks__/indexer.js

```diff
@@ -4,9 +4,9 @@ const db = {
   values: [],
 };
 
-const indexPost = (text, postId) => {
+const indexPost = ({ text, id }) => {
```

Comment:
This doesn't match the signature of the updated `indexPost` you have below:

```js
const indexPost = async ({ text, id, title, published, author }) => {
```

Reply:
Thanks, you are right~, @humphd 
Do you want me to add those parameters as optional ones?
Or just follow the all parameters?

Reply:
We need our mock version (the fake one we use in tests when there's no Elasticsearch service running) to work identically to the real one.  So yes, update the mock so it adds all that data from a `post` object.